dev/com.ibm.websphere.appserver.features/visibility/private/io.openliberty.org.eclipse.microprofile.telemetry1.1.ee-7.0.feature

```diff
@@ -0,0 +1,19 @@
+-include= ~${workspace}/cnf/resources/bnd/feature.props
+symbolicName=io.openliberty.mpTelemetry1.1.ee-7.0
+singleton=true
+-features=\
+  com.ibm.websphere.appserver.mpRestClient-1.0, \
```

Comment:
This one is going to be tricky.  It will be easy to get EE 8 stuff to work, but this will require updates to to mpRestClient 1.0 and 1.1 in order to not require that the public tolerates be carried over into the mpTelemetry public feature.  I would focus on EE 8 and not have EE 7 for this initial PR and then work on EE 7.